pkg/iss/iss.go

```diff
@@ -34,6 +35,22 @@ import (
 	t "github.com/filecoin-project/mir/pkg/types"
 )
 
+// ============================================================
+// Global package variables
+// ============================================================
+
+// Names of modules this protocol depends on.
+// The corresponding modules are expected by ISS to be stored under these keys by the Node.
+var (
```

Comment:
It would be nice for these values to be configurable. Especially `OwnModuleName`, to make it possible to make it possible to run multiple instances of ISS in the same process.

Reply:
I agree.
Not critical for the moment though. Created an issue for that: #99